src/index.ts

```diff
@@ -152,19 +164,29 @@ async function getLinkedIssuesForPr(
       repository?: {
         pullRequest?: {
           closingIssuesReferences: {
-            nodes: { id: string; number: number }[];
+            nodes: {
+              id: string;
+              number: number;
+              repository: {
+                name: string;
+                owner: {
+                  login: string;
+                };
+              };
+            }[];
           };
         };
       };
     };
 
-    const issues = res.repository?.pullRequest?.closingIssuesReferences.nodes;
+    const issues = res.repository?.pullRequest?.closingIssuesReferences.nodes || [];
```

Comment:
I think it makes sense to filter out nodes from other owners here, we do not want to post comments into other orgs repo, I'd say? (we probably can't do that anyhow due to permissions, but it doesn't really make sense I'd say).

Reply:
I think it's valid to do so and if permissions block the creation there's no point in filtering no?